src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -121,6 +123,19 @@ public string InstanceId
             }
         }
 
+        public string ScriptHostInstanceId
```

Comment:
HostInstanceId

Reply:
Also to remove confusion with the existing `ScriptHost.InstanceId` property, I'd say just remove that property, since it is only used one place, and that usage can be replaced with `_scriptSettingsManager.InstanceId`. This makes sense since this ID is a static property of the app, not an instance property of the host.